.travis.yml

```diff
@@ -23,14 +23,18 @@ jobs:
       go: 1.13.x
       name: "Golang-ci Linter"
       script:
+        - make license-check
+        - make bindings
         - make lint
 
     - language: go
       go: 1.13.x
       name: "Run Go tests"
       script:
+        - make bindings
         - SILENT=true make start_eth
         - make deploy_eth
+        - ./scripts/centrifuge/run_chain_ci.sh
```

Comment:
Perhaps if we switch to build tags we can move this and eth to their own jobs, that way we can run all the tests under `./chain` independently of the rest of the bridge.

Reply:
The issue there is that `main.go` still imports both. Will defs look more into what we can do to remove the substrate deps for eth-only bridges